internal/olm/client/client.go

```diff
@@ -280,10 +280,10 @@ func (c Client) printPodErrors(ctx context.Context, depSelectors *metav1.LabelSe
 		return fmt.Errorf("error getting Pods: %v", err)
 	}
 	for _, p := range podList.Items {
-		if p.Status.Phase != corev1.PodSucceeded {
+		if p.Status.Phase != corev1.PodRunning {
 			for _, cs := range p.Status.ContainerStatuses {
 				if !cs.Ready {
-					podErrors[p.Name] = cs.State.Waiting.Message
+					podErrors[p.Name+cs.Name] = cs.State.Waiting.Message
```

Comment:
Do we print this key later? If so, we should probably make it look nicer. E.g.
```suggestion
					podErrors[fmt.Sprintf("%s.%s", p.Name, cs.Name)] = cs.State.Waiting.Message
```"

Reply:
Hi @joelanford  yes, thanks!